libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -429,6 +429,16 @@ bool AC_Fence::pre_arm_check(char *failure_msg, const uint8_t failure_msg_len) c
         hal.util->snprintf(failure_msg, failure_msg_len, "Fences enabled, but none selected");
         return false;
     }
+    
+    // if AUTOENABLE = 1 or 2 warn now, but fail in a later release
+    static uint8_t i = 0 ;
+    i++;
+    if (i > 60) {
```

Comment:
```suggestion
    // PARAMETER_CONVERSION - Added: Jul-2024 for ArduPilot-4.6
    const uint32_t now_ms = AP_HAL::millis();
    if (now_ms - last_autoenable_warn_ms > 60000) {
```

... and add last_autoenable_warn_ms in the header